packages/UsageAnalysis/files/Test Track/Bio/MSA.md

```diff
@@ -0,0 +1,13 @@
+1. Open sample_FASTA.csv.
+2. Add a new column with a formula RandBetween(0,5) - this will be the clusters column.
+3. On the menu ribbon, open **Bio** > **MSA**.
+4. Set Cluster to the new column with a formula (from the step 2).
+5. Check that **Alignment parameters** button adds input parameters to the dialog properly   
```

Comment:
What does it mean 'properly'?

Reply:
@ssSavenko FYI

Reply:
It means that it has right markup, and html elements dont have any visual conflicts

Reply:
than rephrase it in the file